src/parameter/ParameterWidget.cc

```diff
@@ -188,8 +207,33 @@ void ParameterWidget::setComboBoxPresetForSet()
 	}
 }
 
+//set selection
 void ParameterWidget::onSetChanged(int idx)
 {
+	static int lastComboboxIndex(0);
```

Comment:
Not sure I understand the use of `static` here.
e.g. what happens if you have two OpenSCAD windows open with two different files and two customizers?

Reply:
opps... it feels wrong to have a member variable that is only supposed to be used by one function, but I guess a member variable is the best solution after all.